scripts/pull-down-test-api-spec.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+API_SPEC_PATH=../dragonfly
+# Navigate to repo
+cd $API_SPEC_PATH
```

Comment:
Question: Do we need an initial clone?

Reply:
Added a clone if the directory does not exist